bitnami/spark/README.md

```diff
@@ -214,7 +216,8 @@ Go to <https://spark.apache.org/downloads.html> and copy the download url bundli
 
 ```Dockerfile
```

Comment:
Please update the Dockerfile with the following instructions which contain the link updated:

```Dockerfile
FROM bitnami/spark:3.5.0
USER root
RUN install_packages curl
USER 1001
RUN rm -r /opt/bitnami/spark/jars && \
    curl --location https://dlcdn.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz | \
    tar --extract --gzip --strip=1 --directory /opt/bitnami/spark/ spark-3.5.0-bin-hadoop3/jars/
```